### DISCUSSION TITLE
Does Plausible support ClickHouse clusters?

### DISCUSSION THREAD

**Discussion by pschiffe** [plausible/analytics#1603] [Self-Hosted Support] [answered]
Hello,

when Plausible is initializing ClickHouse database, it won't create it on cluster (`CREATE DATABASE xxx ON CLUSTER yyy...`), so I'm wondering whether this setup is supported or not, and whether anyone is using it that way.

Thanks,

peter

**Comment by ukutaht** [comment, marked as answer]
Not as far as I know
